src/components/shared/AddRemoveForm.js

```diff
@@ -75,7 +75,7 @@ class AddRemoveForm extends React.Component {
     }
 
     return availableFilteredItems.map(item => {
-      return <button key={item[labelProp]} className="button" onClick={() => this.addItem(item)}>{utils.codeToText(item[labelProp])}</button>
+      return <button data-test={`item-button`} key={item[labelProp]} className="button" onClick={() => this.addItem(item)}>{utils.codeToText(item[labelProp])}</button>
```

Comment:
`data-test` attributes allow us to easily grab elements while testing. We do this a lot with enzyme:

```
wrapper.find({ 'data-test': 'item-button' })
```

From there, we can count the number of results or check the values of specific items.